umm-spec-lib/src/cmr/umm_spec/migration/version_migration.clj

```diff
@@ -221,6 +221,19 @@
      (dissoc :ComposedOf)
      (dissoc :NumberOfInstruments)))
 
+(defn migrate-tiling-identification-systems
+  "Migrate TilingIdentificationSystems from 1.9 up to 1.10"
```

Comment:
Yes, I agree - we should provide a validation warning (error later) when the TilingIdentificationSystemName isn't valid: CMR-4420.